example/code-snippets/i18n.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react';
+
+import DateRangePicker from 'react-daterange-picker';
+import moment from 'moment';
+
+/**
+ * Load moment locales that you need in your app
+ * to make the localeData available to the DatePicker
+ */
+require('moment/locale/en');
```

Comment:
To make the date picker usable, should we require `moment/locale/en` by default, and set the default proptypes for locale to be `en` - that we we're not making such a backwards compatible change for existing users?